server/sandbox_run.go

```diff
@@ -191,18 +190,7 @@ func getSELinuxLabels(selinuxOptions *pb.SELinuxOption, privileged bool) (string
 			labels = append(labels, "level:"+selinuxOptions.Level)
 		}
 	}
-	var (
-		processLabel, mountLabel string
-		err                      error
-	)
-	processLabel, mountLabel, err = label.InitLabels(labels)
-	if err != nil {
-		return "", "", err
-	}
-	if privileged {
-		processLabel = ""
```

Comment:
this has been completely removed from the code :confused:

Reply:
Added it back.